app/src/main/java/org/fossasia/openevent/general/event/EventService.kt

```diff
@@ -44,6 +50,24 @@ class EventService(
         return eventTopicsDao.getAllEventTopics()
     }
 
+    fun getEventTypes(): Flowable<List<EventType>> {
+        val eventsLocationFlowable = eventTypesDao.getAllEventTypes()
+        return eventsLocationFlowable.switchMap {
+            if (it.isNotEmpty())
+                eventsLocationFlowable
+            else
+                eventTypesApi.getEventTypes()
+                    .map {
+                        eventTypesDao.insertEventTypes(it)
```

Comment:
Don't insert these into DB, let them be fetched from network every time

Reply:
@iamareebjamal okay will revert this